pkg/status/status_test.go

```diff
@@ -191,17 +191,17 @@ func TestProbeLifecycle(t *testing.T) {
 		}
 	}()
 
-	// Dummy handler returning HTTP 500 (it should never be called during probing)
-	dummyRequests := make(chan *http.Request)
-	dummyHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		dummyRequests <- r
+	// Test handler returning HTTP 500 (it should never be called during probing)
```

Comment:
This might be well intentioned, but `test` is super vague.

Reply:
`failingHandler`

Reply:
point taken, I'll revisit!